docs/Wizards_Content.md

```diff
@@ -0,0 +1,36 @@
+# Wizard Content Management Guide
+
+Wizard Management is currently under development.
+
+## How to access the management interface
+
+The Wizard Manager interface can be accessed by using the USAGov tools shortcut on the Drupal Menu Bar, and selecting Wizard Manager.
+
+## Approach
+
+The Wizard Manager allows you to manage wizards in a hierarchical tree view. Wizards are comprised of taxonomy terms that are related to each other in a parent-child relationship. 
+
+## How to create a wizard
+
+In order to create a new wizard, go to the Wizard Manager and press the button "Add term". This step will be used repeatedly in order to create a new wizard and new sub-terms and options of that header wizard. 
+
+On the Add term page, you will be asked and need to fill out the following information:
+
+* Name: Drupal's name for the taxonomy term
+* Language: Language
+* Option Name: This is the name that will show up when you are using the hierarchical tree-view within the Wizard Manager interface
```

Comment:
Actually this field determines what to display in the options list on the term which leads to the term you are creating.